db/postgreSQL/seeds/addReviews.js

```diff
@@ -0,0 +1,75 @@
+const generate = require('../../generate.js');
+const fs = require('fs');
+const ObjectsToCsv = require('objects-to-csv');
+
+exports.seed = async (knex) => {
+
+  console.log('Beginning review data seed...');
+  console.time('seed review time');
+  // query to be used to copy CSV file into table
+  const query = `COPY reviews(user_id, book_id, date, review, rating) FROM '/Users/briankwon/Desktop/hr-immersive/SDC/sdc-brian/reviews.csv' DELIMITER ',' CSV HEADER`;
```

Comment:
I would recommend taking advantage of `__dirname` or the `path` module so you can remove these hardcoded paths that work for only your machine.